news/2018-09-09-project-loved-week-of-september-9th.md

```diff
@@ -0,0 +1,230 @@
+---
+layout: post
+title: "Project Loved: Week of September 9th"
+date: 2018-09-09 21:00:00 +0000
+---
+
+[Project Loved](https://osu.ppy.sh/community/forums/120) returns again with picks in all four modes! Come check out the set, and vote for your favourites!
+
+[![](/wiki/shared/news/banners/project-loved.jpg)](https://osu.ppy.sh/community/forums/120)
+
+Welcome back to [Project Loved](https://osu.ppy.sh/community/forums/120)! Every two weeks, we pick out a few beatmaps from each game mode that will get their shot at making it into the Loved category. From the maps we selected, you can now vote whether or not you want them to be Loved! Click on each maps' header image in this post to be taken to their voting threads.
+
+osu!taiko returns this week from its break, and the team will be selecting four maps per round from now on. They've heard your feedback from the discussion thread that was open for the last two weeks, and will do their best to incorporate it into the picks from now on! Though they didn't make it in time to help out for this week, we're also welcoming two new captains for osu!taiko: [nyanmi-1828](https://osu.ppy.sh/users/6866480) and [-\[ ix Ishida xi \]-](https://osu.ppy.sh/users/242910)!
```

Comment:
"from now on" is repeated. Substitute the first one for "going forward"

Reply:
just removed the first one, not needed really